### DISCUSSION TITLE
对于前后端分离的项目  前端怎么部署

### DISCUSSION THREAD

**Discussion by ftssyang** [apache/apisix#5884] [Q&A]
对于前后端分离的项目  前端怎么部署  可以部署到apisix么  可以给些方案么

**Comment by juzhiyuan** [comment]
Hi @ftssyang, it supports processing static files, but may I know what kind of Web projects you're using? Like Vue.js/ React?

**Reply by ftssyang** [reply]
Vue.js

**Reply by juzhiyuan** [reply]
@ftssyang Would you like to have an online chat here? https://docs.qq.com/doc/DTktRakxqaUFFWndz

https://github.com/apache/apisix/discussions/5873
